BoilerMake/bm-app
# Add live features for BM8

Built day of page at http://localhost:8080/live

To test:
Switch APP STATUS to 4
Create an exec account
Modify the start/end variables at the top of the index.js file to test countdown
Create announcements, and double check that announcements are loading as well as converted properly to EST time zone

TODO still:
- [x] Change / to include a "Go Live" button
- [x] Delete the extra file
- [x] Get design approval
